app/models/drip_email_list.rb

```diff
@@ -0,0 +1,15 @@
+class DripEmailList < ActiveRecord::Base
```

Comment:
Please add to the top of every Ruby file you add `# License: AGPL-3.0-or-later WITH Web-Template-Output-Additional-Permission-3.0-or-later`. We need it for licensing purposes

(Note, it doesn't need to be in the `db/schema.rb`, it's not practical to add there)